Running Maryam with Pyhon 3.12 raises Syntax Warnings

Python 3.12 raises a SyntaxWarning for invalid escape sequences.

Below is an example of running the `crawl_pages` module with Python3.12 instead of `#!/usr/bin/python3`:

```sh
$ maryam -e crawl_pages -d http://localhost
/usr/lib/python3/dist-packages/maryam/core/util/helpers/web_scrap.py:139: SyntaxWarning: invalid escape sequence '\/'
  join = str(join).replace('\/', '/')
/usr/lib/python3/dist-packages/maryam/core/util/iris/cluster.py:34: SyntaxWarning: invalid escape sequence '\w'
  tokens = re.findall("[A-Z]{2,}(?![a-z])|[A-Z][a-z]+(?=[A-Z])|[\'\w\-]+",text)
/usr/lib/python3/dist-packages/maryam/modules/footprint/crawl_pages.py:33: SyntaxWarning: invalid escape sequence '\.'
  'examples': ('crawl_pages -d <DOMAIN> -r "https?://[A-z0-9\./]+"\
/usr/lib/python3/dist-packages/maryam/modules/osint/cloud_storage.py:73: SyntaxWarning: invalid escape sequence '\.'
  + site_url.replace('.','\.')+"/", list( set(LINKS) ) ) ) #escaping . for regex search using replace()
usage: crawl_pages [-h] -d DOMAIN -r REGEX [--more] [-l LIMIT] [--debug] [-t THREAD]
                   [--output] [--api] [--format]
crawl_pages: error: the following arguments are required: -r/--regex
```
Additionally, here are all the affected scripts:
```sh
$ find . -type f -name '*.py' -exec pycodestyle --select W605 {} \;
./maryam/core/util/helpers/web_scrap.py:139:30: W605 invalid escape sequence '\/'
./maryam/core/util/iris/cluster.py:34:67: W605 invalid escape sequence '\w'
./maryam/core/util/iris/cluster.py:34:69: W605 invalid escape sequence '\-'
./maryam/modules/footprint/crawl_pages.py:33:60: W605 invalid escape sequence '\.'
./maryam/modules/osint/cloud_storage.py:73:27: W605 invalid escape sequence '\.'
```

--- a/maryam/core/util/helpers/web_scrap.py
+++ b/maryam/core/util/helpers/web_scrap.py
@@ -136,7 +136,7 @@ class main:
 			if cond1:
 				continue
 
-			join = str(join).replace('\/', '/')
+			join = str(join).replace(r'\/', '/')
 			##########################
 			# ADD OUT SCOPE
 			##########################

--- a/maryam/core/util/iris/cluster.py
+++ b/maryam/core/util/iris/cluster.py
@@ -31,7 +31,7 @@ class main:
 		return [x for x in text if x not in stops]
 
 	def tokenize_and_stem(self, text):
-		tokens = re.findall("[A-Z]{2,}(?![a-z])|[A-Z][a-z]+(?=[A-Z])|[\'\w\-]+",text)
+		tokens = re.findall(r"[A-Z]{2,}(?![a-z])|[A-Z][a-z]+(?=[A-Z])|[\'\w\-]+",text)
 		filtered_tokens = []
 		for token in tokens:
 			if re.search('[a-zA-Z]', token):

--- a/maryam/modules/footprint/crawl_pages.py
+++ b/maryam/modules/footprint/crawl_pages.py
@@ -30,7 +30,7 @@ meta = {
 		('debug', False, False, 'debug the scraper', '--debug', 'store_true', bool),
 		('thread', 1, False, 'The number of links that open per round', '-t', 'store', int),
 	),
-	'examples': ('crawl_pages -d <DOMAIN> -r "https?://[A-z0-9\./]+"\
+	'examples': (r'crawl_pages -d <DOMAIN> -r "https?://[A-z0-9\./]+"\
 	 --output', 'crawl_pages -d <DOMAIN> --limit 2 --more')
 }
 

--- a/maryam/modules/osint/cloud_storage.py
+++ b/maryam/modules/osint/cloud_storage.py
@@ -70,7 +70,7 @@ def module_api(self):
 		self.thread(search, self.options['thread'], engine, query, q_formats, limit, count, meta['sources'])
 			
 		output['links'] += list( self.reglib().filter(r"https?://([\w\-\.]+\.)?"\
-		+ site_url.replace('.','\.')+"/", list( set(LINKS) ) ) ) #escaping . for regex search using replace()
+		+ site_url.replace('.',r'\.')+"/", list( set(LINKS) ) ) ) #escaping . for regex search using replace()
 
 	self.save_gather(output, 'osint/cloud_storage', query, output=self.options.get('output'))
 	return output

